v21.2/create-changefeed.md

```diff
@@ -152,7 +153,7 @@ CockroachDB Type | Avro Type | Avro Logical Type
 
 #### Topic Naming
 
-By default, a Kafka topic has the same name as the table that a changefeed was created on. If a changefeed was created on multiple tables, the changefeed will write to multiple topics corresponding to those table names. You can specify a [topic prefix](#query-parameters) or use the [`full_table_name` option](#options) to modify this.
+By default, a Kafka topic has the same name as the table that a changefeed was created on. If a changefeed was created on multiple tables, the changefeed will write to multiple topics corresponding to those table names. You can specify a [topic prefix](#topic-prefix-param), [an arbitrary topic name](#topic-name-param), or use the [`full_table_name` option](#full-table-option) to modify this.
```

Comment:
feel like we should also mention that you can specify an arbitrary topic name and feed all the tables into that if you want.